_includes/scenarios/integrate/faq.md

```diff
@@ -0,0 +1,7 @@
+Welcome to our FAQ page! Here, you'll find answers to the most common questions about enmeshed. If you're looking for quick and straightforward information, you've come to the right place.
+
+# Common questions
+
+## What is enmeshed?
+
+A description of enmeshed can be found on the [main page]({% link index.md %}#what-is-enmeshed)
```

Comment:
- Sind die Überschriften bzgl. FAQs sinnvoll? (ich glaube mal ja)
- Ist "what is enmeshed" nicht zu generisch -> lieber schon mal mehr technisches Futter, z.B. der Fehler mit dem "RelationshipTemplate not supported, weil RelationshipTemplateContent fehlt"?

Reply:
ich hätte Überschriften für generelle Fragen, technische Fragen zum Backbone, zum Connector und App unterteilt.